### PR TITLE
bots: add libvirt-dbus package in fedora images, fix f27/f28 regressions

### PR DESCRIPTION
**Pull request by martinpitt**
libvirt-dbus package will be needed for cockpit machines.

 * [x] image-refresh fedora-27
 * [x] image-refresh fedora-28
 * [x] image-refresh fedora-i386
 - [x] naughty override for docker regression: https://bugzilla.redhat.com/show_bug.cgi?id=1571736
 - [x] naughty override for FreeIPA leave regression: https://bugzilla.redhat.com/show_bug.cgi?id=1588386
 - [x] naughty override for chrondy SELinux regression: https://bugzilla.redhat.com/show_bug.cgi?id=1584757

### CODE REVIEW

**Comment by cockpituous** [comment]
image-refresh in progress on cockpit-tasks-nblct.
Log: http://fedorapeople.org/groups/cockpit/logs/image-refresh-9285-20180601-094444/

**Comment by cockpituous** [comment]
image-refresh fedora-27 done: https://github.com/cockpituous/cockpit/commits/image-refresh-fedora-27-20180601-103737

**Comment by cockpituous** [comment]
image-refresh in progress on cockpit-tasks-rj3g1.
Log: http://fedorapeople.org/groups/cockpit/logs/image-refresh-9285-20180601-103838/

**Comment by cockpituous** [comment]
image-refresh fedora-28 done: https://github.com/cockpituous/cockpit/commits/image-refresh-fedora-28-20180601-113232

**Comment by cockpituous** [comment]
image-refresh in progress on cockpit-tasks-p9t5t.
Log: http://fedorapeople.org/groups/cockpit/logs/image-refresh-9285-20180601-115050/

**Comment by cockpituous** [comment]
image-refresh fedora-i386 done: https://github.com/cockpituous/cockpit/commits/image-refresh-fedora-i386-20180601-122929

**Comment by martinpitt** [comment]
It seems the docker regression detected in #9267 (https://bugzilla.redhat.com/show_bug.cgi?id=1571736) now also landed in fedora-27, so this needs to land soon. I'll copy it here and rebaes #9267 instead, as the latter seems to need more fixes even.

**Comment by martinpitt** [comment]
[kubernetes testDashboard](https://fedorapeople.org/groups/cockpit/logs/pull-9285-20180604-025858-0af7a0b2-verify-fedora-28/log.html#2) keeps failing like this:
```
Traceback (most recent call last):
  File "/build/cockpit/test/verify/kubelib.py", line 433, in testDashboard
    b.wait_text("#service-list tr[data-name='mock']:first-of-type td.containers", "1")
  File "/build/cockpit/test/common/testlib.py", line 303, in wait_text
    return self.wait_js_func('ph_text_is', selector, text)
  File "/build/cockpit/test/common/testlib.py", line 264, in wait_js_func
    self.wait_js_cond("%s(%s)" % (func, ','.join(map(jsquote, args))))
  File "/build/cockpit/test/common/testlib.py", line 261, in wait_js_cond
    self.raise_cdp_exception("timeout\nwait_js_cond", cond, result["exceptionDetails"], trailer)
  File "/build/cockpit/test/common/testlib.py", line 166, in raise_cdp_exception
    raise Error("%s(%s): %s" % (func, arg, msg))
Error: timeout
wait_js_cond(ph_text_is("#service-list tr[data-name='mock']:first-of-type td.containers","1")): condition did not become true
```

This is apparently another regression in F28 that needs investigating. I can reproduce that locally.

**Comment by martinpitt** [comment]
It's actually the same regression:
```
# docker logs bbdf188cd2e5
container_linux.go:247: starting container process caused "process_linux.go:258: applying cgroup configuration for process caused \"No such device or address\""
```
This also seems to break some container/kubernetes tests.

**Comment by martinpitt** [comment]
[kdump on fedora-i386](https://fedorapeople.org/groups/cockpit/logs/pull-9285-20180604-184949-dbea637b-verify-fedora-i386/log.html#226) also seems to have regressed *sigh*. I retried it several times, investigating locally.

**Comment by martinpitt** [comment]
There just is no pleasing the tests for the refreshed fedora-28 image.. Setting needswork.

**Comment by martinpitt** [comment]
I filed the new SELinux issue as #9328, but the [testIpa](https://fedorapeople.org/groups/cockpit/logs/pull-9285-20180607-025555-dbea637b-verify-fedora-28/log.html#224) failure is different - apparently leaving a domain now fails:
```
Jun 07 04:02:20 x0.cockpit.lan realmd[1857]: Disabling client Kerberos and LDAP configurations
Jun 07 04:02:20 x0.cockpit.lan realmd[1857]: Failed to remove krb5/LDAP configuration: expected str, bytes or os.PathLike object, not NoneType
Jun 07 04:02:20 x0.cockpit.lan realmd[1857]: The ipa-client-install command failed. See /var/log/ipaclient-uninstall.log for more information
Jun 07 04:02:20 x0.cockpit.lan realmd[1857]: process exited: 2753
Jun 07 04:02:20 x0.cockpit.lan realmd[1857]:  ! Running ipa-client-install failed
```
I filed that as known issue #9330 and a downstream report.

File: bots/naughty/fedora-27/9282-docker-cgroup-failure [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Traceback (most recent call last):
```
Comment (allisonkarlitskaya) [inline bots/naughty/fedora-27/9282-docker-cgroup-failure]: According to the downstream bug, this is not an issue on Fedora 27.  Perhaps it makes sense to update that bug with information that F27 is also broken?  Or is this a different issue?

File: bots/naughty/fedora-28/9328-selinux-chronyd-socket [hunk 2]
```diff
@@ -0,0 +1 @@
+Error: audit: type=1400 audit(*): avc:  denied  { sendto } for * comm="chronyd" path="/run/chrony/chrony*.sock"
```
Comment (allisonkarlitskaya) [inline bots/naughty/fedora-28/9328-selinux-chronyd-socket]: It looks like this one has been fixed downstream already.  Should we just rebuild the image?

File: bots/naughty/fedora-27/9282-docker-cgroup-failure (hunk 1 above)
Comment (martinpitt) [reply bots/naughty/fedora-27/9282-docker-cgroup-failure]: No, same issue. I updated the downstream bug.

File: bots/naughty/fedora-28/9328-selinux-chronyd-socket (hunk 2 above)
Comment (allisonkarlitskaya) [reply bots/naughty/fedora-28/9328-selinux-chronyd-socket]: OK.

**Pull request merged**